### DISCUSSION TITLE
Export the list of "Detected dependencies"

### DISCUSSION THREAD

**Discussion by algomaster99** [renovatebot/renovate#18258] [Request Help] [answered]
Renovate has a feature to list all the detected dependencies in the "master issue". The feature was implemented at https://github.com/renovatebot/renovate/pull/15418. Is there a way we can get the list in a machine-processable format like JSON? For example, I would want to get the list of all detected dependencies reported in https://github.com/INRIA/spoon/issues/4143.

**Comment by viceice** [comment, marked as answer]
this is possible, checkout self-hosting docs

**Reply by algomaster99** [reply]
Thank you, @viceice ! I used renovate's npm package (`v32.232.0`) and ran the following command.
```
renovate --token [REDACTED] --dry-run="extract"  --autodiscover --autodiscover-filter "SpoonLabs/sorald" --log-file="renovate.log"
```
I find the list of detected dependencies in the log file generated.
